libs/MailMessage/include/MailAddress.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
I know that `#pragma once` is convenient and I think you know it is not part of the standard, yet you might end up at some point in a project that does not support it so just keep that in mind. I can't find any example online ATM, but I think windriver with C++98/03 didn't work with that, so if you ever encounter it... run.